lls/main.c

```diff
@@ -1074,6 +1242,9 @@ lls_stage2(void *arg)
 		if (ret < 0)
 			return ret;
 		ret = register_nd_router_sol_or_adv_acl_rules(&net_conf->front);
+		if (ret < 0)
+			return ret;
+		ret = register_ping_acl_rules(&net_conf->front);
```

Comment:
This code should go to the `if` statement of ARP, this is the `if` statement of ND.